uamqp/async_ops/client_async.py

```diff
@@ -823,8 +814,8 @@ async def _client_ready_async(self):
                 error_policy=self._error_policy,
                 encoding=self._encoding,
                 desired_capabilities=self._desired_capabilities,
-                loop=self.loop)
-            await asyncio.shield(self.message_handler.open_async(), loop=self.loop)
+                )
+            await asyncio.shield(self.message_handler.open_async(), )
```

Comment:
do we need to replace this with `**self._internal_kwargs`?

Reply:
don't know how I missed so many 🤦